lib/common/repository/auth_repository.dart

```diff
@@ -60,4 +60,12 @@ class AuthRepository {
       rethrow;
     }
   }
+
+  Future<void> logOut() async {
+    try {
+      await _networkUtil.postReq('/logout');
```

Comment:
Add ` await _googleSignIn.signOut();` after this line to logout the Google account.